core/utils/password.go

```diff
@@ -1,78 +1,28 @@
 package utils
 
 import (
-	"errors"
 	"fmt"
-	"regexp"
-	"unicode"
 
+	"github.com/pkg/errors"
 	"go.uber.org/multierr"
 )
 
 const PasswordComplexityRequirements = `
-Must be longer than 12 characters
-Must comprise at least 3 of:
-	lowercase characters
-	uppercase characters
-	numbers
-	symbols
+Must be longer than 16 characters
 Must not comprise:
 	A user's API email
-	More than three identical consecutive characters
 `
 
-var (
-	lowercase = regexp.MustCompile("[a-z]")
-	uppercase = regexp.MustCompile("[A-Z]")
-	numbers   = regexp.MustCompile("[0-9]")
+const RequiredLen = 16
 
-	ErrPasswordMinLength     = errors.New("must be longer than 12 characters")
-	ErrPasswordMinLowercase  = errors.New("must contain at least 3 lowercase characters")
-	ErrPasswordMinUppercase  = errors.New("must contain at least 3 uppercase characters")
-	ErrPasswordMinNumbers    = errors.New("must contain at least 3 numbers")
-	ErrPasswordMinSymbols    = errors.New("must contain at least 3 symbols")
-	ErrPasswordRepeatedChars = errors.New("must not contain more than 3 identical consecutive characters")
+var (
+	ErrPasswordMinLength = errors.Errorf("must be longer than %d characters", RequiredLen)
 )
 
-func countSymbols(password string) (count int) {
-	for _, r := range password {
-		if unicode.IsPunct(r) || unicode.IsSymbol(r) {
-			count++
-		}
-	}
-	return
-}
-
 func VerifyPasswordComplexity(password string) (merr error) {
-	if len(password) <= 12 {
+	if len(password) <= RequiredLen {
```

Comment:
Maybe we should require at least non-repeative symbols?
A password like "aaaaaaaaaaaaaaaa" is safe?

Reply:
I think the consensus is that we don't want to enforce anything at all, it is entirely on the onus of the node operator